Maquetado/scss/styles.scss

```diff
@@ -147,3 +149,21 @@ body {
 .footer {
   background-color: $blue;
 }
+
+@media (max-width: 1024px) {
+  .tools-container {
+    display: inline;
+    margin-left: -100px;
+  }
+
+ .main-title {
+    display: none;
+  }
+
+ .techno {
+    border: none;
+    border-bottom: 2px solid $green;
```

Comment:
This way, every tech will have the border bottom, what you actually need to do is use the `:not()` and `:last-child` pseudo classes to direct the style to those elements :smile: :

```scss
.tools-container {
  &:not(:last-child) {
    border-right: none;
    border-bottom: ...
    ...
  }
}
```